src/ServerProviders/Nginx/Plugins.ServerProviders.Nginx/ServerProviderNginx.cs

```diff
@@ -81,7 +82,8 @@ private string GetShellCommandOutput(string command)
             }
             else
             {
-                command = $"{command.Replace("\"", "\\\"")}";
+                // TODO: It would be best if we could configure the location of nginx for Windows, similar to NginxManager does for the config path
+                command = $"/C \"{command.Replace("\"", "\\\"").Replace("nginx", "C:\\nginx\\nginx.exe")}\""; ;
```

Comment:
what is the `/C` doing?

Reply:
"/c carries out the command specified by <string> and then exits the command processor."
https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/cmd#parameters

I saw it mentioned in several places as best practice to use when passing cmd.exe parameters via CLI. Without it, the Process hangs and never exits, which is probably why you had to add the timeout int to WaitForExit() in your previous testing. It wasn't until I added this parameter that I started seeing an exit code returned by Process.

Without /c
![image](https://github.com/webprofusion/certify-plugins/assets/14862506/ca85f994-fc49-4029-a631-b23c15a0420d)

With /c
![image](https://github.com/webprofusion/certify-plugins/assets/14862506/77b6aaaf-8bb8-4f74-81e3-c70f49d8597d)